### ISSUE TITLE
Lily-Pad Jesus

### ISSUE THREAD

**Issue by lilZoeyXanax** [ImpactDevelopment/ImpactIssues#1968]
**Is your feature request related to a problem? Please describe.**
A lot of server have anti cheat, and a lot of servers jesus doesnt work.

**Describe the solution you'd like**
Have the normal jesus, and dolphin mode, but also add a mode where if we have lily-pads in our hotbar it'll auto place under our feet so we can still walk on water with anticheats without having to pay attention to placing it every jump.

**Additional context**
I couldnt find anything else related to lily pads and jesus being used together. It would be amazing to have this and would allow us to easily traverse water without having to pay attention to placing lily pads under our feet.

**Comment by GreenMan36** [comment]
This would be great but isn't this a kind of scaffold?

**Comment by S-B99** [comment]
This is literally just scaffold

**Comment by lilZoeyXanax** [comment]
> This is literally just scaffold

Yes and no, Scaffold doesn't work with lily pads, at least I cant get it to work with lily pads, and scafold will always build under you even if you jump. This not only takes up a lot of blocks, but also you'd have to turn scafolding on and off to go down layers If there was a lily pad jesus there would be two benefits: Lily pads are very easy to collect, and the major difference is, that you'd be able to spring jump (and other forms of speed), staying on the same Y level and jesus placing the lily pad perfectly under you moments before hitting the water. As lily pads can be placed on solid water, blocks cant. (It doesn't work with liquid interact either.)

**Comment by S-B99** [comment]
Then just rename this issue to "scaffold support Lily pads"

**Comment by lilZoeyXanax** [comment]
But scaffold implies being able to build vertically, jesus is designed for covering water, if you're using scaffold and have lily pads in your inventory and go over water trying to climb in Y level, you'll have to remove it from your hotbar, jesus implies going over water. Which is why it would make more sense to be a jesus module.

**Comment by S-B99** [comment]
No it wouldn't. Just don't go up...

**Comment by lilZoeyXanax** [comment]
but if you wanna sprint jump on the water then you'd go up

**Comment by S-B99** [comment]
This is honestly the most super specific issue I've seen in so long. Just code your own module with some open source client. Nobody is going to add something super specific that 3 people will use

**Comment by GreenMan36** [comment]
I think this would be nice. 

Combine parcour to sprint jump on the edge of the pad, then place and do the same when landing with scaffold. Modules are there, just need some tweaking from someone that knows how.
